src/depfile_parser_test.cc

```diff
@@ -133,7 +133,7 @@ TEST_F(DepfileParserTest, Escapes) {
   // it through.
   string err;
   EXPECT_TRUE(Parse(
-"\\!\\@\\#$$\\%\\^\\&\\[\\]\\\\:",
+"\\!\\@\\#$$\\%\\^\\&\\[\\]\\\\ :",
```

Comment:
Not sure about the space. Shouldn't : at the end be special and therefore not need any escaping?

Reply:
Just added a new commit. I used lookahead to make sure that the next character is not whitespace nor EOF. As whitespace in side of filenames are escaped this should work I believe. Sadly not an expert on re2c